File: 128-longest-consecutive-sequence/longest-consecutive-sequence.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {
    int n=nums.size();
    sort(nums.begin(),nums.end());
    if(n==0) return 0;
    int cnt=1;
    int maxC=1;
    for(int i=0;i<n-1;i++){
        if(nums[i]+1==nums[i+1]){
            cnt++;
            maxC=max(maxC,cnt);
        }else if(nums[i]==nums[i+1]){
            continue;
        }else{
            cnt=1;
        }
    }
    return maxC;
    }
};

File: 128-longest-consecutive-sequence/longest-consecutive-sequence.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        sett=set(nums)
        maxi=0
        for n in sett:
            if n-1 not in sett:
                cnt=0
                while n+cnt in sett:
                    cnt+=1
                maxi=max(maxi,cnt)

        return maxi